apps/frontend/src/pages/tasks/libs/helpers/convert-category-name-to-css-class-name/convert-category-name-to-css-class-name.helper.ts

```diff
@@ -0,0 +1,5 @@
+const convertCategoryNameToCssClassName = (categoryName: string): string => {
```

Comment:
```suggestion
const convertCategoryNameToKebabCase = (categoryName: string): string => {
```
Let's not limit to css classNames